qml/DeviceManagerViewer.qml

```diff
@@ -212,6 +215,34 @@ PingPopup {
                             text: connection.typeToString() + " " + connection.createConfString()
                         }
 
+                        // Use same layout if item is not visible, to have a proper alignment
+                        Item {
+                            height: parent.height
+                            width: height
+                            PingImage {
+                                id: iconConfiguration
+                                source: StyleManager.settingsIcon()
+                                anchors.fill: parent
+                                selected: deviceConfigureMouseArea.containsMouse
+                                Layout.alignment: Qt.AlignRight
+
+                                visible: {
+                                    return connection.deviceType() == PingEnumNamespace.PingDeviceType.PING360 &&
+                                        connection.type() == AbstractLinkNamespace.Udp
```

Comment:
does it mean that the option will show when connected via the companion bridge? Can we also check if the port is 12345?

Reply:
The idea is to allow any UDP connection, why is there a need to check for the port ?

Reply:
these messages will not get to the Ping360 if it is connected to the companion via serial and using bridges

Reply:
The gear icon will appear but won't work

Reply:
No, that's not the case and it's not related to the UDP port but to the subnet network.
I'll add a warning message for it in the configuration.

Reply:
> these messages will not get to the Ping360 if it is connected to the companion via serial and using bridges

I got it now thanks!

Reply:
Done via check of isCompanion